front/app/containers/DisabledAccount/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import moment from 'moment';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import { Title } from 'components/smallForm';
+import { useSearchParams } from 'react-router-dom';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+export default () => {
+  const [searchParams, _] = useSearchParams();
```

Comment:
```suggestion
  const [searchParams] = useSearchParams();
```

I saw _ was not used, so I removed it.